tests/qam-kgraft/update_kgraft.pm

```diff
@@ -99,15 +116,32 @@ sub run() {
 
     #run HEAVY Load script
     script_run("bash /tmp/heavy_load.sh");
+
     # warm up system
     sleep 15;
 
     die "Patch isnt valid for this system" if is_patch_needed($patch);
 
+    #die if patch is already installed on the system
```

Comment:
not needed , it duplicates line 123 and comment says 100 and 101 returncode but check is for 101/102